packages/tools/client-debugger/devtools/src/packageVersion.ts

```diff
@@ -6,4 +6,4 @@
  */
 
 export const pkgName = "@fluid-experimental/devtools";
-export const pkgVersion = "2.0.0-internal.4.2.0";
+export const pkgVersion = "2.0.0-internal.4.3.0";
```

Comment:
Probably want to revert this. I have a fix up here: #15517

Reply:
Thanks for catching that! Reverted :)